app/extensions/safe/auth-web-app/components/app_list.js

```diff
@@ -104,10 +98,10 @@ export default class AppList extends Component {
 
   getNoAppsContainer() {
     return (
-      <div className="no-apps">
+      <div key={"noApps"} className="no-apps">
```

Comment:
Is the `key` really required in places like this one? assuming this is not part of a list of elements, if not, perhaps we shouldn't add them in this type of places to keep the code simpler and cleaner?

Reply:
Noted, I will check and confirm in a bit.